src/modules/blockchain-events/implementation/ot-ethers/ot-ethers.js

```diff
@@ -119,11 +96,11 @@ class OtEthers extends BlockchainEventsService {
             const allContracts = [...contractsAray, ...assetStoragesArray];
 
             for (const [contractName, contractAddress] of allContracts) {
-                if (CONTRACT_EVENTS_LISTENED.includes(contractName) && ABIs[contractName] != null) {
+                if (MONITORED_CONTRACTS.includes(contractName) && ABIs[contractName] != null) {
                     this.contracts[blockchain][contractName] = new ethers.Contract(
                         contractAddress,
                         ABIs[contractName],
-                        this.providers[blockchain],
+                        provider,
```

Comment:
Can we not tie provider and contract, if we have multiple rpcs for robustness, one of them fails, the whole contracts can't be pulled even thou we have different working rpc?